perf/evd/classic/evd_classic_perf.cpp

```diff
@@ -24,8 +24,12 @@ std::vector<std::string> tol_based_names = {
 };
 
 int main() {
```

Comment:
Since we have different executable for all benchmarks and reading the inputs from STDIN now, I think adding these lines might be beneficial to get faster experiment results:

```cpp
std::ios_base::sync_with_stdio(false);
std::cin.tie(0);
```

Reply:
Good point! I will update the code.